### DISCUSSION TITLE
Install in VS-Code

### DISCUSSION THREAD

**Discussion by cyberfish** [viperet/vscode-sftp-revived#2] [General]
Hi, how can I install your extension in VS-Code? 
Many thanks.
cf

**Comment by RattyBiscuit** [comment]
On the releases page (https://github.com/viperet/vscode-sftp-revived/releases) you need to download the `.vsix` file which can then be installed from the Extensions menu:

![image](https://user-images.githubusercontent.com/7528696/126772448-bb31d834-b637-4978-88c3-0cc1f40671ed.png)
